reference/forms/types/repeated.rst

```diff
@@ -129,6 +129,11 @@ the label::
         'second_options' => ['label' => 'Repeat Password'],
     ]);
 
+.. note::
```

Comment:
What about line 100 instead?

Reply:
For me this is most logic place to have a note about options. I would like to have a note about this exactly at the place where documentation describes options for inner fields.

Reply:
I think this information. is too important and should be shown before any options. This is needed in order to handle validation, and the line I like is about that section.

Reply:
Okay. Done.

Reply:
Thanks, I leave that one open for now.